src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -357,20 +361,27 @@ private void UpdateLayoutModels()
             }
         }
 
-        private void ParseDeviceInfoData()
+        private void ParseDeviceInfoData(bool debug = false)
         {
             try
             {
-                FileStream inputStream = File.Open(Settings.ActiveZoneSetTmpFile, FileMode.Open);
-                var jsonObject = JsonDocument.Parse(inputStream, options: default).RootElement;
+                string layoutType = "blank";

```

Comment:
Add a comment that this is a non localizable string.

Reply:
Don